OWML.ModLoader/Owo.cs

```diff
@@ -90,7 +91,7 @@ private void OnLogMessageReceived(string message, string stackTrace, LogType typ
         {
             if (type == LogType.Error || type == LogType.Exception)
             {
-                _console.WriteLine($"Unity log message: {message}. Stack trace: {stackTrace?.Trim()}");
+                _console.WriteLine($"Unity log message: {message}. Stack trace: {stackTrace?.Trim()}", MessageType.Error);
             }
```

Comment:
should we allow all Unity messages through, with matching message types? It might be spammy 🤷

Reply:
it will be *extremely* spammy 🥴 
there's a lot of repeated log messages by mobius